source/tutorials/kubernetes.rst

```diff
@@ -180,71 +350,143 @@ Configure and run CRI-O + kata-runtime
+      [crio.runtime]
+      manage_network_ns_lifecycle = true
 
-**Notes about flannel add-on**
+   On |CL| systems this is done automatically.
 
-If you choose the `flannel` add-on, then you must add the following to the :command:`kubeadm init` command:
+#. Once the cluster initialization is complete, continue reading about how to
+   `Use your cluster`_.
 
-.. code-block:: console
+Use your cluster
+****************
 
-   --pod-network-cidr 10.244.0.0/16
+Once your master control plane is successfully initialized, follow the
+instructions presented about how to use your cluster and its *IP*, *token*,
+and *hash* values are displayed. It is important that you record this
+information because it is required to join additional nodes to the cluster.
 
-Furthermore, if you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
+A successful initialization looks like this:
 
 .. code-block:: console
 
-    [crio.runtime]
-    manage_network_ns_lifecycle = true
+   Your Kubernetes control-plane has initialized successfully!
 
-Use your cluster
-****************
+   To start using your cluster, you need to run the following as a regular user:
 
-Once your master control plane is successfully initialized, instructions on how to use your cluster and its *IP*, *token*, and *hash* values are displayed. It is important that you record the cluster values because you need them to join worker nodes to the cluster. Some values have a valid period. The values are presented in a format similar to:
+   mkdir -p $HOME/.kube
+   sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+   sudo chown $(id -u):$(id -g) $HOME/.kube/config
 
-.. code-block:: bash
+   ...
 
-   kubeadm join <master-ip>:<master-port> --token <token> --discovery-token-ca-cert-hash <hash>
+   You can now join any number of machines by running the following on each node
+   as root:
 
+   kubeadm join <control-plane-host>:<control-plane-port> --token <token> --discovery-token-ca-cert-hash sha256:<hash>
 
-**Congratulations!**
 
-You've successfully installed and set up Kubernetes in |CL| using CRI-O and kata-runtime. You are now ready to follow on-screen instructions to deploy a pod network to the cluster and join worker nodes with the displayed token and IP information.
+With the first node of the cluster setup, you can continue expanding the
+cluster with additional nodes and start deploying containerized applications.
+For further information on using Kubernetes, see `Related topics`_. 
 
-Related topics
-**************
+.. note:: 
 
-Read the Kubernetes documentation to learn more about:
+   By default, the master node does not run any pods for security reasons. To
+   setup a single-node cluster and  allow the master node to also run pods,
```

Comment:
Id change to  "To setup a single-node cluster and  allow the master node to also run pods, the master node will need its taints removed. See the ..."